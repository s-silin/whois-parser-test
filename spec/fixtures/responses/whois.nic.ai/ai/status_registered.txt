Domain Name: google.ai
Domain ID: 325702_nic_ai
WHOIS Server: whois.nic.ai
Updated Date: 2018-02-28T23:55:33.196Z
Creation Date: 2017-12-16T05:37:20.801Z
Registry Expiry Date: 2019-09-25T05:37:20.856Z
Domain Status: clientTransferProhibited https://icann.org/epp#clientTransferProhibited
Domain Status: clientUpdateProhibited https://icann.org/epp#clientUpdateProhibited
Domain Status: clientDeleteProhibited https://icann.org/epp#clientDeleteProhibited
Registrar: Markmonitor
Registrar Abuse Phone: +1.2083895740
Registrar Abuse Email: ccops@markmonitor.com
Registrant ID: 526771_nic_ai
Registrant Name: Domain Administrator
Registrant Organization: Google LLC
Registrant Street: 1600 Amphitheatre Parkway
Registrant City: Mountain View
Registrant State/Province: CA
Registrant Postal Code: 94043
Registrant Country: US
Registrant Phone: +1.6502530000
Registrant Fax: +1.6502530001
Registrant Email: dns-admin@google.com
Admin ID: 526771_nic_ai
Admin Name: Domain Administrator
Admin Organization: Google LLC
Admin Street: 1600 Amphitheatre Parkway
Admin City: Mountain View
Admin State/Province: CA
Admin Postal Code: 94043
Admin Country: US
Admin Phone: +1.6502530000
Admin Fax: +1.6502530001
Admin Email: dns-admin@google.com
Billing ID: 514507_nic_ai
Billing Name: CCOPS Billing
Billing Organization: MarkMonitor Inc.
Billing Street: 3540 East Longwing Lane
Billing Street: Suite 300
Billing City: Meridian
Billing State/Province: Idaho
Billing Postal Code: 83646
Billing Country: US
Billing Phone: +1.2083895740
Billing Fax: +1.2083895771
Billing Email: ccopsbilling@markmonitor.com
Tech ID: 526771_nic_ai
Tech Name: Domain Administrator
Tech Organization: Google LLC
Tech Street: 1600 Amphitheatre Parkway
Tech City: Mountain View
Tech State/Province: CA
Tech Postal Code: 94043
Tech Country: US
Tech Phone: +1.6502530000
Tech Fax: +1.6502530001
Tech Email: dns-admin@google.com
Name Server: ns1.zdns.google
Name Server: ns2.zdns.google
Name Server: ns3.zdns.google
Name Server: ns4.zdns.google
DNSSEC: unsigned
>>> Last update of WHOIS database: 2018-07-25T01:42:03.152Z <<<

TERMS OF USE: You are not authorized to access or query our WHOIS database through the use of electronic processes that are high-volume and automated.  THis WHOIS database is provided by as a service to the internet community.

The data is for information purposes only. We do not guarantee its accuracy. By submitting a WHOIS query, you agree to abide by the following terms of use: You agree that you may use this Data only for lawful purposes and that under no circumstances will you use this Data to: (1) allow, enable, or otherwise support the transmission of mass unsolicited, commercial advertising or solicitations via e-mail, telephone, or facsimile; or (2) enable high volume, automated, electronic processes that apply to CoCCA it's members (or CoCCA or member computer systems). The compilation, repackaging, dissemination or other use of this Data is expressly prohibited.
