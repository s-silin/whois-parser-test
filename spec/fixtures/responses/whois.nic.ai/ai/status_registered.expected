#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %TIME{2017-12-16T05:37:20Z}

#updated_on
  %s %TIME{2018-02-28T23:55:33Z}

#expires_on
  %s %TIME{2019-09-25T05:37:20Z}


#nameservers
  %s %CLASS{array}
  %s %SIZE{4}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1.zdns.google"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns2.zdns.google"
  %s[2] %CLASS{nameserver}
  %s[2].name == "ns3.zdns.google"
  %s[3] %CLASS{nameserver}
  %s[3].name == "ns4.zdns.google"
